templates/observability/what-is-observability.html

```diff
@@ -1,251 +1,273 @@
+          Controllability:
+          <br />
+          observability in action
+        </h2>
+      </div>
+      <div class="col">
+        <p>
+          <b>The forgotten twin to observability</b>
+        </p>
+        <p>
+          The concept of observability originates in the field of modern control system theory, and its formulation has well withstood the test of time. And it was not a standalone concept either. Rather, observability had a twin concept, called controllability, which can be roughly defined as: "the property of a system to regulate itself and related systems to reliably produce the correct outputs given the provided inputs."
+        </p>
+        <p>
+          Nowadays, controllability is not a term many software engineers use daily, but its essence is embodied, for example, by operators <a href="https://jaas.ai/?_ga=2.74388370.1830430303.1624442919-1608689799.1623932549">implemented with Juju</a> or other frameworks, which steer the software they operate and the infrastructure underneath to provide seamless configuration management and achieve scalability, reliability and graceful degradation.
+        </p>
+        <div class="p-cta-block">
+          <a href="/blog/tag/model-driven-operations"
+             class="p-button"
+             aria-label="Learn more about model-driven operations">Learn more about model-driven operations</a>
         </div>
+      </div>
     </div>
-</section>
+  </section>
 
-<section class="p-strip">
-    <div class="row u-vertically-center">
-        <div class="col-5 u-hide--medium u-hide--small u-hide--medium u-hide--small u-align--center">
-            {{ image (
-                url="https://assets.ubuntu.com/v1/e68fd355-1+discovery_AW.svg",
-                alt="",
-                width="403",
-                height="230",
-                hi_def=True,
-                loading="auto"
-                ) | safe
-            }}
-        </div>
-        <div class="col-7">
-            <h2>Ready to make your applications observable?</h2>
-            <p>Canonical can support your open source observability stack with security fixes and LTS support, as well as run your best-of-breed, open source monitoring tools reliably and at scale.</p>
-            <a class="p-button--positive" href="/observability">More about open source observability</a>
+  <section class="p-section--deep">
+    <hr class="p-rule is-fixed-width" />
+    <div class="row--50-50">
+      <div class="col">
+        <h2>
+          Ready to make
+          <br />
+          your applications observable?
+        </h2>
+      </div>
+      <div class="col">
+        <p>
+          Canonical can support your open source observability stack with security fixes and LTS support, as well as run your best-of-breed, open source monitoring tools reliably and at scale.
+        </p>
+        <div class="p-cta-block">
+          <a href="https://docs.google.com/document/u/0/d/1NxJdT3BD6EWnpsu-eathNyranQVImnDGhuCjzsIWh4s/edit"
```

Comment:
This should link to the bubble's index page. In the copy doc it's pointing to the google doc too, but it's nonsense.

Reply:
Thanks for the correction! @akbarkz